clifford/_mvarray.py

```diff
@@ -9,14 +9,38 @@
 call_array = np.vectorize(MultiVector.__call__)
 
 
+def _interrogate_nested_mvs(input_array):
```

Comment:
```suggestion
def _interrogate_nested_mvs(input_array) -> Tuple[Tuple[int], Layout, np.dtype]:
```
You'll need to import `Layout` too.